instance/templates/instance/ansible/vars.yml

```diff
@@ -103,6 +127,22 @@ EDXAPP_TIME_ZONE: 'UTC'
 EDXAPP_ENV_EXTRA:
   LANGUAGE_CODE: 'en'
 
+# Secret keys
+# TODO: Set these to random values (see OC-1700).
```

Comment:
@antoviaque We weren't coordinating. I was just copying these from sample vars without thinking about the effect they would have if it gets into production. I am away next week so I would like to get this merged before OC-1700. I think I'll just remove these variables from my PR and add paste the deleted code in a comment on OC-1700 in case its helpful.